generator/computers_c.py

```diff
@@ -0,0 +1,69 @@
+import sys
+sys.path.append("ParseCAN")
+import os
+import ParseCAN
+from common import computer_c_dir_path, coord, templ, ifndef, endif, frame_handler
+
+
+def msg_handler(frame, name_prepends, num_tabs, fw):
+    tot_name = name_prepends + '_' + frame.name
+    fw(
+        '\t' * num_tabs + 'case CANlib_{}_key:\n'.format(tot_name) +
+        '\t' * (num_tabs + 1) + 'handle_{}_msg(&frame);\n'.format(tot_name) +
+        '\t' * (num_tabs + 1) + 'break;\n'
+    )
+
+
+def write(can, computers, output_path=computer_c_dir_path):
+    os.makedirs(output_path, exist_ok=True)
```

Comment:
`pathlib` is better but if it works don't fix it I guess.

Reply:
Unless it doesn't work on Windows lmao

Reply:
This works on Windows, so I'm just gonna resolve this